src/TaskRunner/Commands/ToolCommands.php

```diff
@@ -399,14 +394,18 @@ protected function componentInsecure($modules, $packages)
                 foreach ($insecurePackages as $insecurePackage) {
                     $historyTerms = $this->getPackageDetails($insecurePackage['name'], $insecurePackage['version'], '8.x');
                     $packageInsecureConfirmation = true;
-                    $msg = "Package " . $insecurePackage['name'] . " have a security update, please update to an safe version.";
+                    $msg = "Package {$insecurePackage['name']} have a security update, please update to a safe version.";
 
-                    if (!in_array("insecure", $historyTerms['terms'])) {
-                        $packageInsecureConfirmation = false;
-                        $msg = $msg . " (Confirmation failed, ignored)";
+                    if (empty($historyTerms['terms'])) {
+                        $this->say("Something went wrong when checking the package {$insecurePackage['name']}");
```

Comment:
If the query to d.org to fetch the release history fails then we do not block the execution. Looks it is missing a  $packageInsecureConfirmation = false; just after line 400